src/edn/mod.rs

```diff
@@ -780,15 +719,6 @@ mod test {
     use alloc::vec;
 
     use super::*;
-    #[test]
```

Comment:
do we have these tests covered elsewhere?

Reply:
I removed them because with `(i64, u64)` they're infallable